### DISCUSSION TITLE
How are the green boxes decided on the contribution history?

### DISCUSSION THREAD

**Discussion by Ice57Block** [community/community#116608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How many contributions do I have to make to get the green box tick on my contribution history?
I am trying to keep up my daily activity and make sure I have a bright green contribution history!

**Comment by ethanw2457** [comment]
The number of contributions needed is relative to the other number of contributions you've made I believe. So if you have made a lot of contributions in a day, then it will most likely take a lot or similar number of contributions to reach bright green.
